tests/quantization/test_precision_init.py

```diff
@@ -418,16 +421,15 @@ def test_hawq_broadcast_avg_traces_in_distributed_mode(tmp_path):
 ]
 
 
-@pytest.mark.parametrize('hw_config', [None, HWConfigType.VPU],
-                         ids=['no_constraints', 'vpu_constraints'])
```

Comment:
Why was this parametrization removed?

Reply:
As discussed, this test originaly had a bug.  It shouldn't work with VPU config.
Done.